webapp/src/redux/ducks/studyMaps.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-param-reassign */
```

Comment:
Only on the concerned line.
You can also remove destructuration of `drafState` to remove the errors.

Reply:
even with destructuration the eslint warning persists unfortunalty.. I can add it on both lines but I find this repetitive 
didn't find how to shut it without disabling this rule, do you know an alternative maybe ?

Reply:
You have a warning with `draftState.xxx = ...`?